libs/viewer/include/viewer/Settings.h

```diff
@@ -213,6 +213,7 @@ struct ViewerOptions {
     float cameraFocalLength = 28.0f;
     float cameraFocusDistance = 10.0f;
     bool autoScaleEnabled = true;
+    bool autoInstancingEnabled = true;
```

Comment:
The doesn't match its Java corollary and should probably default to "false" since it doesn't work with morphing / skinning.

Reply:
ok, will set to false.